api/vote.go

```diff
@@ -150,9 +149,30 @@ func (a *API) newVote(w http.ResponseWriter, r *http.Request) {
 	// for example, if the process is not in this sequencer, the vote will be
 	// rejected
 	if ok, err := a.storage.ProcessIsAcceptingVotes(pid); !ok {
-		ErrProcessNotAcceptingVotes.WithErr(err).Write(w)
+		if err != nil {
+			ErrProcessNotAcceptingVotes.WithErr(err).Write(w)
+			return
+		}
+		ErrProcessNotAcceptingVotes.Write(w)
 		return
 	}
+	// check if the address has already voted, to determine if the vote is an
+	// overwrite or a new vote, if so check if the process has reached max
+	// voters
+	isOverwrite, err := state.HasAddressVoted(a.storage.StateDB(), process.ID, process.StateRoot, vote.Address.BigInt())
+	if err != nil {
+		ErrProcessNotFound.Withf("could not open state: %v", err).Write(w)
```

Comment:
Is this error correct? ProcessNotFound does seems to be the correct thing to return here